From 4d4d5c8c3503871d715c2cec6c131adfaa0013b2 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 11 Aug 2014 12:26:39 -0400
Subject: [mfd] lpc_ich: Change Avoton to iTCO v3

Message-id: <1407759999-16143-9-git-send-email-prarit@redhat.com>
Patchwork-id: 87603
O-Subject: [RHEL7.1 PATCH BZ 1084134 8/8] mfd: lpc_ich: Change Avoton to iTCO v3
Bugzilla: 1084134
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1084134

commit c48cf59878685cc06b71bb2a3ca17b61103c8de7
Author: Peter Tyser <ptyser@xes-inc.com>
Date:   Mon Mar 10 16:34:56 2014 -0500

    mfd: lpc_ich: Change Avoton to iTCO v3

    The register layout of the Avoton is compatible with the iTCO v3
    register layout.

    Signed-off-by: Peter Tyser <ptyser@xes-inc.com>
    Tested-by: Rajat Jain <rajatjain@juniper.net>
    Reviewed-by: Guenter Roeck <linux@roeck-us.net>
    Signed-off-by: Lee Jones <lee.jones@linaro.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/mfd/lpc_ich.c b/drivers/mfd/lpc_ich.c
index 8f78606..ea1dcef 100644
--- a/drivers/mfd/lpc_ich.c
+++ b/drivers/mfd/lpc_ich.c
@@ -502,7 +502,7 @@ struct lpc_ich_info lpc_chipset_info[] = {
  },
  [LPC_AVN] = {
   .name = "Avoton SoC",
-  .iTCO_version = 1,
+  .iTCO_version = 3,
   .gpio_version = AVOTON_GPIO,
  },
  [LPC_COLETO] = {
-- 
1.7.1